scripts/misc/commit_checker.sh

```diff
@@ -65,16 +66,31 @@ do
         cat scripts/misc/commit_message_recom.txt
         exit 4
     fi
-    if [ ${current_line_len} -gt 72 ] ; then
-        echo ""
-        echo "ERROR: ${current_line_len} chars in ${line}-th line is too long. Any line length must be less than or equal to 72 chars !"
-        echo ""
-        cat commit.log
-        echo "----------------------------------------------"
-        echo "Please read following rules:"
-        cat scripts/misc/commit_message_recom.txt
-        exit 3
+    if [[ ${current_line} == *"Signed-off-by"* ]]; then
+        echo " ${line}-th line is sign-off line."
+        if [ ${current_line_len} -gt 80 ]; then
```

Comment:
the whole 'if' block looks like a function that can be called twice with 80 and with 72